### DISCUSSION TITLE
Welcome to dotnet-podcasts Discussions!

### DISCUSSION THREAD

**Discussion by jamesmontemagno** [microsoft/dotnet-podcasts#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by intesar112** [comment]
Is there a YouTube explanation for this project?

**Comment by kamal-jeet-singh** [comment]
Can any please help me how we can play mp3 or other format file from app  local  e.g Resource or www folder.
